runtime/parachains/src/paras.rs

```diff
@@ -457,6 +457,12 @@ pub mod pallet {
 	pub(super) type CodeByHash<T: Config> =
 		StorageMap<_, Identity, ValidationCodeHash, ValidationCode>;
 
+	/// Scraped on chain data for extracting resolved disputes as well as backing votes.
+	#[pallet::storage]
+	#[pallet::getter(fn on_chain_votes)]
+	pub(crate) type OnChainVotes<T: Config> =
```

Comment:
`Paras` really doesn't feel like the right module for this. `ParasInherent` would make more sense.